DicomRtImportExport/Logic/vtkSlicerDicomRtImportExportModuleLogic.cxx

```diff
@@ -831,7 +831,19 @@ bool vtkSlicerDicomRtImportExportModuleLogic::vtkInternal::LoadExternalBeamPlan(
     unsigned int dicomBeamNumber = rtReader->GetBeamNumberForIndex(beamIndex);
     const char* beamName = rtReader->GetBeamName(dicomBeamNumber);
     unsigned int nofCointrolPoints = rtReader->GetBeamNumberOfControlPoints(dicomBeamNumber);
-
+    const char* beamType = rtReader->GetBeamType(dicomBeamNumber);
+    const char* treatmentDeliveryType = rtReader->GetBeamTreatmentDeliveryType(dicomBeamNumber);
+    bool notStaticBeam = true;

```

Comment:
I'd rename this to isStaticBeam and put a ! in front of the strcmp. Negating variables etc just introduce confusion.

Reply:
fixed